src/linagora.esn.unifiedinbox/app/components/mailbox-shared-settings/mailbox-shared-settings.controller.spec.js

```diff
@@ -43,12 +44,19 @@ describe('The InboxMailboxSharedSettingsController controller', function() {
         return user;
       })
     };
+
+    esnAuth = {
+      signInCompletePromise: {
+        then: callback => callback()
+      }
```

Comment:
same here, you just need to return a resolved promise

```
signInCompletePromise: $q.when()
```

Reply:
I tried, but it did not work, I can still try again to make sure.

Reply:
Ok, it works, so I replaced it.